src/EditorFeatures/TestUtilities/Diagnostics/AbstractUserDiagnosticTest.cs

```diff
@@ -140,8 +151,16 @@ protected Document GetDocumentAndAnnotatedSpan(TestWorkspace workspace, out stri
                 // Simple code fix.
                 foreach (var diagnostic in diagnostics)
                 {
+                    if (fixer == null)
+                    {
+                        result.Add(Tuple.Create(diagnostic, (CodeFixCollection)null));
```

Comment:
This was added so the test infrastructure can handle diagnostics without fixers.  Otherwise, it was crashing.